electron_app/DownloadManager.ts

```diff
@@ -0,0 +1,73 @@
+const Downloader = require('./Downloader');
+const path = require('path');
+const { handleFolder, fileAlreadyExists } = require('./helpers/handleDisk');
+const downloadStats = require('./helpers/downloadInfo').downloadStats;
+const handleEvents = require('./helpers/handleEvents');
+
+let filePath = '';
```

Comment:
The files are now .ts but it loks like they're not really converted completely yet. E.g. `filePath` and `fileExists` should be local variables and `allDownloads` should be a field in `DownloadManagerClass`. Also the `any`s should be migrated to use types.